doc/content/msfr/griffin_pgh_results.md

```diff
@@ -1,27 +1,160 @@
+shown next.
+
+!media media/msfr/Nek_Re40000_vfield.png
+    style=width:30%
+    caption=Nek5000 Velocity magnitude field for Re 40K.
+
+!media media/msfr/PH_Re40000_vfield.png
+    style=width:30%
+    caption=Pronghorn Velocity magnitude field for Re 40K.
+
+!media media/msfr/Nek_Re40000_pfield.png
+    style=width:30%
+    caption=Nek5000 Pressure field for Re 40K.
+
+!media media/msfr/PH_Re40000_pfield.png
+    style=width:30%
+    caption=Pronghorn Pressure field for Re 40K.
+
+The parameter $\delta$ for this simulation is set to 0.12. This is the result
+of a parametric study focused on minimizing the error of the velocity field,
+specifically at the minimum core radius.The velocity magnitude at the minimum
+core radius is plotted in the following figure.
+
+!media media/msfr/Velocity_LinePlot_Re40K.png
+    style=width:60%
+    caption=Velocity magnitude at the minimum core radius.
+
+The behavior of the velocity fields are similar between both codes. The
+momentum diffusion at the core of the reactor is relatively higher in Pronghorn.
+This comes as a result of the limitation in the zero-equation turbulence model,
+which assumes a constant mixing length in the bulk region.
+
+Pronghorn's total simulation time is 4 minutes using 4 cores.
+
+## Results for Reynolds number 1,000,000
+
+The boundary conditions for this problem are velocity inlet profile, pressure
+outlet, symmetry at the center of the geometry along the X axis and standard
+velocity wall functions at the walls. Time derivatives are added to the momentum
+equation for relaxation purposes.
+
+An exponentially decaying viscosity ramp is used for convergence purposes.
+
+\begin{equation}
+  \mu_{sim} = \mu_{phys} \cdot (exp(-3 \cdot t) + 1)
+\end{equation}
+
+Velocity magnitude and pressure fields obtained in Nek5000 and Pronghorn are
+shown next.
+
+!media media/msfr/Nek_Re1M_vfield.png
+    style=width:30%
+    caption=Nek5000 Velocity magnitude field for Re 1M.
+
+!media media/msfr/PH_Re1M_vfield.png
+    style=width:30%
+    caption=Pronghorn Velocity magnitude field for Re 1M.
+
+!media media/msfr/Nek_Re1M_pfield.png
+    style=width:30%
+    caption=Nek5000 Pressure field for Re 1M.
+
+!media media/msfr/PH_Re1M_pfield.png
+    style=width:30%
+    caption=Pronghorn Pressure field for Re 1M.
+
+The parameter $\delta$ for this simulation is set to 0.1. This is the result
+of a parametric study focused on minimizing the error of the velocity field,
+specifically at the minimum core radius.The velocity magnitude at the minimum
+core radius is plotted in the following figure.
+
+The behavior of the velocity fields are similar between both codes. The
+momentum diffusion at the core of the reactor is slightly higher in Pronghorn.
+
+!media media/msfr/Velocity_LinePlot_Re1M.png
+    style=width:60%
+    caption=Velocity magnitude at the minimum core radius.
+
+Pronghorn's total simulation time is 15 minutes using 4 cores.
```

Comment:
say where time was spent. initialization?

Reply:
This is only navier stokes, so it begins from zero velocity field.

Reply:
It's the total simulation time